client/src/listings/Listings.js

```diff
@@ -41,28 +48,40 @@ function Listings(props) {
   const [subcategoryId, setSubcategoryId] = useState('');
   const [categoryOptions, setCategoryOptions] = useState([]);
   const [subcategoryOptions, setSubcategoryOptions] = useState({});
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(15);
+  const [pageNum, setPageNum] = useState(0);
```

Comment:
This should be renamed to a more clear variable name, maybe `totalPages` or `pageCount`.